pom.xml

```diff
@@ -37,6 +38,14 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <!-- Import dependency management from Spring Boot -->
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>${spring.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
```

Comment:
Is there a special reason for adding this block? I've verified that even without this block of code the default Spring version is `1.5.22.RELEASE` anyway. Is it a good idea to use the default version of Spring? wdyt?

Reply:
Yep, it was there because in the first changes I used org.apache.camel:camel-bom:2.21.0.fuse-760027-redhat-00001 as suggested by Fuse team, but later I changed again to fuse dependency. So you are right, that dependency on spring-boot can be removed.